lib/tasks/thredded_tasks.rake

```diff
@@ -5,4 +5,14 @@ namespace :thredded do
   end
 
   task :nuke, [:slug] => :destroy
+
+  namespace :install do
+    desc 'Copy emoji to the Rails `public/emoji` directory'
+    task :emoji do
+      require 'emoji'
```

Comment:
It certainly does. Here's the crappy part - it places the images in the wrong place. I originally just wrapped the original task in the thredded namespace and let it rip but after trying that, the images ended up in `public/images/emoji` but then referenced them in `public/emoji`.